calico_node/startup/startup.go

```diff
@@ -524,11 +526,26 @@ func ipv6Supported() bool {
 	return supported
 }
 
+// defineNATOutgoing checks the CALICO_NAT_OUTGOING parameter used for
+// default IPPool Creation. It returns a boolean.
+
+func defineNATOutgoing () bool {
+	switch strings.ToLower(os.Getenv("CALICO_NAT_OUTGOING")) {
```

Comment:
I think this name should be `CALICO_IPPOOL_NAT_OUTGOING` if there is only one variable for both IPv4 and IPv6.  If, as I suspect, that we should have 2 then it should match the format of the other default pool configuration variables.